src/einsteinpy/coordinates/utils.py

```diff
@@ -6,6 +6,28 @@
 _c = constant.c.value
 
 
+class Error(Exception):
+    def __init__(self, message):
+        self.message = message
+
+    def __str__(self):
+        return self.__class__.__name__
+
+    def __repr__(self):
+        return self.__class__.__name__ + " : " + self.message
+
+
+class CoordinateError(Error):
+    def __init__(self, message):
+        self.message = message
+
+    def __str__(self):
+        super().__str__()
```

Comment:
Redefining `__str__` and `__repr__` in `CoordinateError` is redundant (after putting `super()` in the `__init__`).